src/odemis/acq/align/test/roi_autofocus_test.py

```diff
@@ -39,12 +41,12 @@ class RoiAutofocusTestCase(unittest.TestCase):
     def setUpClass(cls):
         testing.start_backend(METEOR_CONFIG)
 
-        # create some streams connected to the backend
-        # cls.microscope = model.getMicroscope()
         cls.ccd = model.getComponent(role="ccd")
         cls.focus = model.getComponent(role="focus")
         cls.stage = model.getComponent(role="stage")
 
+        cryoSwitchSamplePosition(FM_IMAGING).result()
```

Comment:
Can you add a comment explaining why this is needed? I think that's because in this mode, the focus position will be near its "good" position and so the simulator will simulate (almost) in focused images, so the autofocus will work .